shared/src/db/database.ts

```diff
@@ -626,6 +630,51 @@ class database extends Dexie {
             });
     }
 
+    private deleteExpired() {
+        console.log(`isCMS: ${cmsFlag.cms}`);
+
+        if (cmsFlag.cms) {
+            console.log("CMS is on");
+            return;
+        } else {
+            console.log("CMS is not on...deleting documents");
+            const now = DateTime.now().toMillis();
+
+            this.docs
+                .where("type")
+                .anyOf(DocType.Content, DocType.Post)
```

Comment:
Only content can expire, posts don't have an expiry date (we'll have to see how we handle that, maybe also delete  posts without content?)